assemblies/rosa-managing-worker-nodes.adoc

```diff
@@ -0,0 +1,14 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-managing-worker-nodes"]
+:context: rosa-worker-nodes
+
+= Managing worker nodes
+
+[role="_abstract"]
+This section describes how to manage worker nodes with {product-title}.
+
+You can edit the configuration of worker nodes for options such as scaling, instance type, labels, and taints. The majority of changes for worker nodes are configured on machine pools. A _machine pool_ is a group of worker nodes in a cluster that have the same configuration, which provides ease of management.
```

Comment:
flip the order of these sentences and perhaps link to OCP docs on machinepools?

Reply:
We can't link to OCP docs right now. But I did flip the order of the sentences.